doc/release/1.4.0-notes.rst

```diff
@@ -127,6 +127,159 @@ Other changes
 Authors
 =======
 
+* 30blay +
+* @endolith
+* @hugovk
+* Abhinav +
+* Anne Archibald
+* ashwinpathak20nov1996 +
+* Nelson Auner +
+* aypiggott +
+* Christoph Baumgarten
+* Peter Bell
+* Sebastian Berg
+* Bharat123rox +
+* Arman Bilge +
+* Benedikt Boecking +
+* Christoph Boeddeker +
+* brandondavid +
+* Daniel Bunting
+* Evgeni Burovski
+* Angeline Burrell +
+* Angeline G. Burrell +
+* CJ Carey
+* Chelsea +
+* chelsea.l +
+* Mak Sze Chun +
+* Christian Clauss +
+* Stephen P Cook +
+* Dylan Cutler +
+* Anirudh Dagar +
+* Aidan Dang +
+* daniloagst +
+* dankleeman +
+* Brandon David +
+* Tyler Dawson +
+* DerWeh +
+* domengorjup +
+* Joe Driscoll +
+* Dávid Bodnár
+* Fletcher Easton +
+* etienne +
+* Johann Faouzi
+* Yu Feng
+* Isuru Fernando +
+* Matthew H Flamm
+* Martin Gauch +
+* Gabriel Gerlero +
+* Gina +
+* Ralf Gommers
+* Chris Gorgolewski +
+* Edouard Goudenhoofdt +
+* Jan Gwinner +
+* Matt Haberland
+* hadshirt +
+* Pierre Haessig +
+* David Hagen
+* Charles Harris
+* Alex Henrie +
+* Francisco J. Hernandez Heras +
+* Andreas Hilboll
+* L Hiltner +
+* Lindsey Hiltner
+* Thomas Hisch
+* Min ho Kim +
+* Huize +
+* Gert-Ludwig Ingold
+* jakobjakobson13 +
+* JDkuba +
+* Todd Jennings
+* He Jia
+* jonathanconroy +
+* jwd0023 +
+* Andrew Knyazev +
+* Holger Kohr +
+* Mateusz Konieczny +
+* krzysztofpioro +
+* Philipp Lang +
+* Eric Larson
+* Antony Lee
+* Gregory R. Lee
+* Jesse Livezey
+* lobpcg +
```

Comment:
@lobpcg Looks like I need to remap to your name above

Reply:
Andrew Knyazev